grasa_event_locator/templates/messaging/provider_account_approval_mail.txt

```diff
@@ -3,4 +3,4 @@ Hello,
 Your account for {{ user.org_name }} at the GRASA Event Locator is approved!
 Please login below to start adding events:
 
-http://grasa.larrimore.de/login.html
+{{ config.site_name.prefix }}{{ config.site_name.fqdn }}/login.html
```

Comment:
here is where we should be manually adding the "://".  could be something like 
{{ config.site_name.protocol }}://{{ config.site_name.fqdn }}/login.html

Reply:
Resolved. See commit e219d5b001f40c67489551690e581815b1ba7f28.